lib/streamlit/string_util.py

```diff
@@ -40,14 +36,29 @@ def clean_text(text: "SupportsStr") -> str:
     return textwrap.dedent(str(text)).strip()
 
 
+def _contains_special_chars(text: str) -> bool:
```

Comment:
@LukasMasuch If I understand correctly this code, this method could be replaced with python built-in 
[str.isalnum](https://docs.python.org/3/library/stdtypes.html#str.isalnum) method